components/HamburgerIcon.tsx

```diff
@@ -0,0 +1,45 @@
+import { Icon, createIcon } from '@chakra-ui/react'
+
+const HamburgerIcon = () => (
+    <Icon width="30" height="30" viewBox="0 0 30 30" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <rect width="6" height="6" fill="white"/>
+    <rect width="6" height="6" fill="white"/>
+    <rect width="6" height="6" fill="white"/>
+    <rect width="6" height="6" fill="white"/>
+    <rect y="12" width="6" height="6" fill="white"/>
+    <rect y="12" width="6" height="6" fill="white"/>
+    <rect y="12" width="6" height="6" fill="white"/>
+    <rect y="12" width="6" height="6" fill="white"/>
+    <rect y="24" width="6" height="6" fill="white"/>
+    <rect y="24" width="6" height="6" fill="white"/>
+    <rect y="24" width="6" height="6" fill="white"/>
+    <rect y="24" width="6" height="6" fill="white"/>
+    <rect x="12" width="6" height="6" fill="white"/>
+    <rect x="12" width="6" height="6" fill="white"/>
+    <rect x="12" width="6" height="6" fill="white"/>
+    <rect x="12" width="6" height="6" fill="white"/>
+    <rect x="12" y="12" width="6" height="6" fill="white"/>
+    <rect x="12" y="12" width="6" height="6" fill="white"/>
+    <rect x="12" y="12" width="6" height="6" fill="white"/>
+    <rect x="12" y="12" width="6" height="6" fill="white"/>
+    <rect x="12" y="24" width="6" height="6" fill="white"/>
+    <rect x="12" y="24" width="6" height="6" fill="white"/>
+    <rect x="12" y="24" width="6" height="6" fill="white"/>
+    <rect x="12" y="24" width="6" height="6" fill="white"/>
+    <rect x="24" width="6" height="6" fill="white"/>
+    <rect x="24" width="6" height="6" fill="white"/>
+    <rect x="24" width="6" height="6" fill="white"/>
+    <rect x="24" width="6" height="6" fill="white"/>
+    <rect x="24" y="12" width="6" height="6" fill="white"/>
+    <rect x="24" y="12" width="6" height="6" fill="white"/>
+    <rect x="24" y="12" width="6" height="6" fill="white"/>
+    <rect x="24" y="12" width="6" height="6" fill="white"/>
+    <rect x="24" y="24" width="6" height="6" fill="white"/>
+    <rect x="24" y="24" width="6" height="6" fill="white"/>
+    <rect x="24" y="24" width="6" height="6" fill="white"/>
+    <rect x="24" y="24" width="6" height="6" fill="white"/>
```

Comment:
We can clean this up in a few ways:

1. We can remove the `fill` properties from each `rect` and set the color using `fill` on `Icon`
2. There are a lot of duplicate `rect` elements that we don't need. There should only be 9

```suggestion
  <Icon
    width="30"
    height="30"
    viewBox="0 0 30 30"
    fill="white"
    xmlns="http://www.w3.org/2000/svg"
  >
    <rect width="6" height="6" />
    <rect y="12" width="6" height="6" />
    <rect y="24" width="6" height="6" />
    <rect x="12" width="6" height="6" />
    <rect x="12" y="12" width="6" height="6" />
    <rect x="12" y="24" width="6" height="6" />
    <rect x="24" width="6" height="6" />
    <rect x="24" y="12" width="6" height="6" />
    <rect x="24" y="24" width="6" height="6" />
```